src/ucm/cuda/cudamem.c

```diff
@@ -1,5 +1,6 @@
 /**
  * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+ * Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
comma needed?
 if we inline with other format? year after organization ?

Reply:
This is consistent with how the copyright reads in other files for NVIDIA contributions. Also, I don't have an issue changing this to look like the entry above but NVIDIA guideline does have the year(s) followed by a comma.